frappe/build.py

```diff
@@ -235,6 +235,9 @@ def watch(apps=None):
 	if apps:
 		command += " --apps {apps}".format(apps=apps)
 
+	if frappe.conf.autoreload_on_build:
+		command += " --auto-reload"
```

Comment:
`--autoreload` for consistency with the console command. Should have an envvar for the same.

I'm not sure about the `autoreload_on_build` key name. 🥲

Reply:
Happy to accept suggestions for the config name.

Reply:
`live-reload` ?

That's common name for this behaviour ;)

So someone searching "frappe live reload" will land on it.

Reply:
@gavindsouza
`reload_on_build` / `refresh_on_build`?  I guess "auto" is implied by `on_build`.
Also, the console flag made sense as `autoreload` instead of `auto-reload` as the IPython plugin used was called `autoreloader`

@ankush +1 for `live-reload`

Reply:
I'd prefer `live-reload` too. I think it would be more apt to make the console command's flag as `--live-reload` too. 😅